documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -446,46 +589,71 @@ The value of an _update_ change event on this collection will actually have the
+* Names of the collection and database that contain the updated document.
+* If the event was part of a snapshot.
+* Timestamp for when the event occurred and ordinal of the event within the timestamp.
+* Unique identifier of the MongoDB operation, which depends on the version of MongoDB. It is either the `h` field in the oplog event, or a field named `stxnid`, which represents the `lsid` and `txnNumber` fields from the oplog event.
 
-* The `op` field value is now `u`, signifying that this document changed because of an update
-* The `patch` field appears and has the stringified JSON representation of the actual MongoDB idempotent change to the document, which in this example involves setting the `first_name` field to a new value
-* The `filter` field appears and has the stringified JSON representation of the MongoDB selection criteria used for the update
-* The `after` field no longer appears
-* The `source` field structure has the same fields as before, but the values are different since this event is from a different position in the oplog
-* The `ts_ms` shows the timestamp that {prodname} processed this event
+|===
 
 [WARNING]
 ====
-The content of the patch field is provided by MongoDB itself and its exact format depends on the specific database version.
-You should therefore be prepared for potential changes to the format when upgrading the MongoDB instance to a new version.
-
-All examples in this document were obtained from MongoDB 3.4 and might differ if you use a different one.
+In a {prodname} change event, MongoDB provides the content of the `patch` field. The format of this field depends on the version of the MongoDB database. Consequently, be prepared for potential changes to the format when you upgrade to a newer MongoDB database version. Examples in this document were obtained from MongoDB 3.4, In your application, event formats might be different.
 ====
 
 [NOTE]
 ====
-Update events in MongoDB's oplog do not have the _before_ or _after_ states of the changed document, so there's no way for the connector to provide this information.
-However, because _create_ or _read_ events _do_ contain the starting state, downstream consumers of the stream can actually fully-reconstruct the state by keeping the latest state for each document and applying each event to that state. {prodname} connector's are not able to keep such state, so it is not able to do this.
+In MongoDB's oplog, _update_ events do not contain the _before_ or _after_ states of the changed document. Consequently, it is not possible for a {prodname} connector to provide this information. However, MongoDB's oplog contains a document's starting state in _create_ and _read_ events. Downstream consumers of the stream can reconstruct document state by keeping the latest state for each document and comparing the state in a new event with the saved state. {prodname} connector's are not able to keep this state. 
```

Comment:
I think the following is slightly misleading:
```
MongoDB's oplog contains a document's starting state in _create_ and _read_ events.
```
For _read_ events, we don't consult the oplog but rather we stream the contents of the actual collection directly, just like we stream rows directly from the base tables for relational databases.  I think it might be sufficient to reword the above to something like:
```
However, the {prodname} connector can provide starting state for _create_ and _read_ events.
```

Reply:
@Naros - can or always does? Is this correct: 

However, a {prodname} connector provides a document's starting state in _create_ and _read_ events.

Reply:
I just committed an update with the wording I suggested. Let me know if something needs to change.

Reply:
LGTM, thanks!